Unity/CHANGELOG.md.meta

```diff
@@ -0,0 +1,7 @@
+fileFormatVersion: 2
```

Comment:
Why is this meta file included in the changes? (Why does it exist at all?) Likewise for most of the .meta files in this PR.

Reply:
it is required by unity. if it isnt present, the file will not be acknowledged. the .md file will be placed next to the meta file